### PR TITLE
Win32 stdcall

### PR DESCRIPTION
**Pull request by cielavenir**
(let me test win32 for a while)

### CODE REVIEW

**Comment by cielavenir** [comment]
Not WIP anymore; ready for review now.

File: src/libbz2.jl [hunk 1]
```diff
@@ -58,55 +58,109 @@ function compress_init!(stream::BZStream,
                         blocksize100k::Integer,
                         verbosity::Integer,
                         workfactor::Integer)
+  if is_windows() && Sys.WORD_SIZE==32
```
Comment (bicycle1885) [inline src/libbz2.jl]: I'd rather suggest defining a constant like `const WIN32 = is_windows() && Sys.WORD_SIZE == 32` and use it anywhere needed.

**Review by bicycle1885 — COMMENTED** [review]
Looks cool! I added a comment but you may reject it if you disagree with me.

File: src/libbz2.jl (hunk 1 above)
Comment (cielavenir) [reply src/libbz2.jl]: Great. I'll do this later

**Comment by bicycle1885** [comment]
Thank you!

**Pull request merged**